ds/fact_evaluation7.py

```diff
@@ -292,3 +297,18 @@ def aux_convert_dj1(cod):
     if cod == 'DJ1':
         return -1
     return cod
+
+def add_ciclo(ano):
+    # ENADE evaluation cycle
+    ### Cycle I = 2004, 2007, 2010, 2013, 2016
+    ### Cycle II = 2005, 2008, 2011, 2014, 2017
+    ### Cycle III = 2006, 2009, 2012, 2015
+    if (ano == 2004) or (ano == 2007) or (ano == 2010) or \
+    (ano == 2013) or (ano == 2016):
+        return 'Ciclo 1'
+    elif (ano == 2005) or (ano == 2008) or (ano == 2011) or \
+    (ano == 2014) or (ano == 2017):
+        return 'Ciclo 2'
+    elif (ano == 2006) or (ano == 2009) or (ano == 2012) or \
+    (ano == 2015):
+        return 'Ciclo 3'
```

Comment:
Since the new column is named 'CICLO', wouldn't be better for this function to return integers?